modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -22,6 +22,8 @@
 
 class NestedTabs extends Widget_Nested_Base {
```

Comment:
@mykytamurzin @davseve 

Previously we echo-ed the icons without escaping. I have studied the rendering, and I concluded that it was safe to do so.

Reason: 
Inside the Icons Manager we use this function to get the SVG file: https://github.com/elementor/elementor/blob/b5285f0a0a4bb368b9a9dfa1fc52304b4b663aa3/includes/managers/icons.php#L309-L308
And the get SVG file function, sanitizes the SVG file before rendering it: 
https://github.com/elementor/elementor/blob/87144ba01bee0361b09d8a84df92cf8d8f640a31/core/files/file-types/svg.php#L229-L228.

But still I think it isn't the cleanest method to echo an icon with an phpcs comment.

In this PR I have removed the echo function to render the SVG file, and I have rendered the SVG directly inside the markup. I believe this is a cleaner way to do it.

Disadvantage though: I needed to quite a large PR with many changes to change the rendering methods of the icons, titles and containers.

Can you please study this and give your opinion. If this is it worth it, then can you please review my PR?